pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -122,11 +150,16 @@ def __enter__(self):
     @property
     def cmd(self):
 
-        cmd = [
-            'pytest',
+        if self.django_toxenv:
+            cmd = ['tox']
+            cmd += ['-e', self.django_toxenv]
+            cmd += ['--']
+        else:
+            cmd = ['pytest']
+        cmd.extend([
             '--ds={}'.format('{}.envs.{}'.format(self.root, self.settings)),
             '--junitxml={}'.format(self.report_dir / "nosetests.xml"),
-        ] + self.test_options_flags
+        ] + self.test_options_flags)
```

Comment:
Why mix `+=` with `extend()` for list extension?

Reply:
That's the way the code was before - I'll fix.